### ISSUE TITLE
Invoke constraint popup on constraint creation

### ISSUE THREAD

**Issue by hlorus** [hlorus/CAD_Sketcher#259]
Allows to quickly enter a dimension value when adding a dimensional constraint.
Ideally allows to directly enter a number.

**Comment by amrsoll** [comment]
A question about this; I can see a smal dialog that shows on the bottom of the screen when creating a constraint.

Which part of the code is responsible for creating it? Would that window qualify for punching in the numbers if we can give focus to it?

![image](https://user-images.githubusercontent.com/24782867/186448317-dc59235e-85ac-48d1-812b-a79a8a7e9bbc.png)

**Comment by hlorus** [comment]
Afaik that's called the operator redo panel. An operator can define a draw method which will be used to draw that panel. There's likeley no support to give it focus, therefor i don't think that's the right approach.

**Comment by amrsoll** [comment]
So we we want a pop up right next to the entity/constraint we create in the 3D view?

Example from fusion 360
![image](https://user-images.githubusercontent.com/24782867/188287889-97241a61-e446-42e1-86ce-5eea3cc6941d.png)

**Comment by hlorus** [comment]
Yep, well basically just invoke the menu that appears when you click on a constraint.

**Comment by amrsoll** [comment]
Founds the proper thing we want to use from the documentation: Panels (and not menus)

https://docs.blender.org/api/current/bpy.types.Panel.html

Which you can spawn using [`Operator.draw()`](https://docs.blender.org/api/current/bpy.types.Operator.html#custom-drawing)

I'll try making a prototype this weekend if nobody else is tackling this :)

**Comment by hlorus** [comment]
I've tried implementing the functionality already, spawning the menu isn't a big deal, the bigger problem is to activate the value property so users could directly enter a number. IMO that's needed otherwise it just feels annoying.

**Comment by hlorus** [comment]
I've posted a bug report at blender regarding the activate_init issue:
https://developer.blender.org/T101113

**Comment by amrsoll** [comment]
I guess we could do a basic text parser for now if only text input work with `activate_init`, might also be helpful for people who want to put units in the input, but that would be secondary .

**Comment by hlorus** [comment]
> I guess we could do a basic text parser for now if only text input work with `activate_init`,

True but i wouldn't recommend such a workaround, this task is really just a quality of life improvement.

> might also be helpful for people who want to put units in the input, but that would be secondary .

That's already supported with Integer/Float properties.

**Comment by DariusJDebugs** [comment]
I'd like to claim the bounty for this issue. Estimated development time: ~1–2 hours.

The solution is complete and has been submitted as a pull request.

I implemented direct numeric value entry during dimensional constraint creation and validated the completed change.

I would like the bounty for completion of #259.
